plantcv/plantcv/visualize/sizes.py

```diff
@@ -0,0 +1,76 @@
+# Visualize an annotated image with object sizes
+
+import os
+import cv2
+import random
+import numpy as np
+# from plantcv.plantcv import find_objects
+from plantcv.plantcv import params
+from plantcv.plantcv import color_palette
+from plantcv.plantcv._debug import _debug
+
+
+def sizes(img, mask, num_objects=100):
```

Comment:
I'm thinking `pcv.visualize.sizes` might not quite be descriptive enough? What about `pcv.visualize.obj_sizes` maybe?

Reply:
`pcv.visualize.obj_sizes` looks good to me

Reply:
Agreed!